apps/data-pipeline/websoc-scraper/src/lib.ts

```diff
@@ -637,7 +641,9 @@ const doDepartmentUpsert = async (
     await tx
       .insert(websocSectionMeetingToLocation)
       .values(
-        Array.from(meetingsToLocations.entries())
+        meetingsToLocations
+          .entries()
+          .toArray()
           .map(([meetingId, v]) => {
```

Comment:
Sorry I'm dumb and missed this :facepalm: This can be `map` then `toArray` as well